### ISSUE TITLE
[DatePicker] [[DateRangePicker]]: On accessing date pickers through screen reader, corresponding ‘Day’ and ‘week count’ information is not read

### ISSUE THREAD

**Issue by I506645** [SAP/ui5-webcomponents-react#6058]
### Describe the bug

On accessing date pickers through screen reader, corresponding ‘Day’ and ‘week count’ information is not read. 

Example: On accessing Date: ‘19’, Month: ‘April’ and Year: ‘2019’ within the date picker, screen reader reads it as ‘April 19, 2019’.

### Reproduction steps

Access the DatePicker or DateRangePicker component using Screen-reader.

### Expected Behaviour

Screen reader should read all the information available on the screen.

On accessing all the dates with the date picker, corresponding ‘Day’ and ‘week count’ information should also be read.

Example: On accessing Date: ‘19’, Month: ‘April’ and Year: ‘2019’ within the date picker, it should be read as ‘Week 16, Friday, April 19, 2019’.

### UI5 Web Components for React Version

1.17.3

### UI5 Web Components Version

1.17.0

### Browser

Chrome

### Operating System

Windows

### Declaration

- [X] I’m not disclosing any internal or sensitive information.
